docs/source/private-data/private-data.md

```diff
@@ -150,7 +157,7 @@ documentation on [transaction flow](../txflow.html).
    transaction and the block. Upon validation/commit, the private data is moved to
    their copy of the private state database and private writeset storage. The
    private data is then deleted from the `transient data store`.
-
+   
```

Comment:
We've also talked about having a Note after sections like this, to mention that client application can gather the endorsements instead of delegating that to a target gateway peer, along with a link to the corresponding release-2.3 doc topic that explains the flow from that perspective. But, we could add add that in a later PR so that the Note is consistent across the various topics.

Reply:
Note: The client application can collect the endorsements instead of delegating that step to the target peer. Refer to the [v2.3 Peers and Applications](https://hyperledger-fabric.readthedocs.io/en/release-2.3/peers/peers.html#applications-and-peers) topic for details.